clients/js/README.md

```diff
@@ -68,6 +68,44 @@ const provider = sapphire.wrap(window.ethereum);
 window.ethereum = sapphire.wrap(window.ethereum); // If you're feeling bold.
 ```
 
+### Viem
```

Comment:
Should we keep or replicate these snippets to https://github.com/oasisprotocol/docs? Or is this okay for now? cc @matevz

Reply:
Or I have my answer in https://github.com/oasisprotocol/docs/issues/585

Reply:
@CedarMist we probably want BOTH our npm user facing documentation as well as official documentation snippets right?

Reply:
In addition to the npm readme, the WAGMI part should go into https://docs.oasis.io/dapp/sapphire/browser and the Viem part into https://docs.oasis.io/dapp/sapphire/quickstart or perhaps https://docs.oasis.io/dapp/sapphire/guide.

btw. Does Viem support encrypted transactions using our wrapper? I thought we only integrate with ethers hooks?

Reply:
> In addition to the npm readme, the WAGMI part should go into https://docs.oasis.io/dapp/sapphire/browser and the Viem part into https://docs.oasis.io/dapp/sapphire/quickstart or perhaps https://docs.oasis.io/dapp/sapphire/guide.

This sure is more convenient to reference on a PR now. 😉

> Does Viem support encrypted transactions using our wrapper? I thought we only integrate with ethers hooks?

I'm able to get [this](https://explorer.oasis.io/mainnet/sapphire/tx/0xf8636d5ee8511e92addb6eb839a6216485d2504bfaadd0b5c8cf6afe4a4b9611) through as opposed to unwrapped [transaction](https://explorer.oasis.io/mainnet/sapphire/tx/0x8ba5586611a2e8753d0d6030f489be466537aa7e8c73d928fed54a1ec30c5975), and I sampled these demo [examples](https://github.com/wevm/viem/tree/4517ae49eb9f910688dfdc880e80dbf0d13565be/examples) as working.

It is all EIP-1193 the way down with [custom](https://viem.sh/docs/clients/transports/custom.html) providers the way they [wrap](https://github.com/wevm/viem/blob/4517ae49eb9f910688dfdc880e80dbf0d13565be/src/clients/transports/custom.ts#L30-L47) it.